### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"

### DISCUSSION THREAD

**Discussion by Shawn88-ops** [community/community#17687] [Copilot Conversations] [answered]
I'm a newcomer to the Github Copilot recently.Today, I tried to use this plug-in on VScode，But it didn't work and gave me such a mistake：GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"
![image](https://user-images.githubusercontent.com/57652481/171682367-174c3aa4-967c-4f69-934b-d929004ce454.png)
How can I solve it?
（My English is not very good，if you can,Better reply in Chinese，thank you）

**Comment by Shawn88-ops** [comment]
I have tried to uninstall and reload GitHub Copilot,or change its version,but none of them worked.It seems that the problem occurred when I first installed it to verify my identity and log in to the GitHub account, but I didn't find the function to try to log in again. What should I do now?

**Reply by Shawn88-ops** [reply]
I finally found a way to log in again, but it still doesn't work.The version of VScode is 1.67.2,and the Copilot is 1.25.6056.I'm getting a little upset about myself.Can anyone help me?

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by Shawn88-ops** [reply]
Um,I think it's unlikely,because my GitHub account login works well, and the connection between VScode and GitHub also seems to work well. I think that if SSL traffic is really intercepted, at least the connection between VScode and GitHub will not work properly.
I just tried again to log out of my account and log in again, but the trouble hasn't changed.

> Um,I think it's unlikely,because my GitHub account login works well, and the connection between VScode and GitHub also seems to work well. I think that if SSL traffic is really intercepted, at least the connection between VScode and GitHub will not work properly. I just tried again to log out of my account and log in again, but the trouble hasn't changed.

**Comment by davecheney** [comment, marked as answer]
Something has either corrupted the set of SSL roots on your machine or a third party is intercepting your ssl connections.

**Reply by Shawn88-ops** [reply]
Thank you,but how can I solve it?

**Reply by Shawn88-ops** [reply]
Umm. I don't know what had happened, but it works well now, maybe it's because I turned off the accelerator connected to GitHub. Anyway, Thank you for your help!

**Comment by 3070720260** [comment]
我最近也碰到了这个问题，把加速器关了就行

**Reply by Mant1e** [reply]
为什么我不行呢

**Comment by FranklinDX3906** [comment]
大佬，后面整通了吗，我也遇到了相同的问题，辛辛苦苦把学生认证弄好了，结果原来可以用，学生认证了之后就不行了

**Reply by Mant1e** [reply]
整好了，先退出VScode，再关掉加速器，然后再开

**Comment by FranklinDX3906** [comment]
感谢，我也整好了，是因为开了fastgithub的缘故



---Original---
From: ***@***.***&gt;
Date: Sun, Aug 21, 2022 17:07 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] GitHub Copilot could not connect to server.Extension activation failed: "unable to verify the first certificate"(Discussion #17687)




 
整好了，先退出VScode，再关掉加速器，然后再开

**Reply by wirsnow** [reply]
感谢，fastgithub一关，idea之类都可以用了
